matter_server/server/sdk.py

```diff
@@ -128,10 +117,9 @@ async def commission_with_code(
         node_id: int,
         setup_payload: str,
         discovery_type: DiscoveryType,
-    ) -> PyChipError:
+    ) -> int:
```

Comment:
So this is fixed now ? Because previously this called returned that PyChipError (which was strange enough on its own)

Reply:
Yeah the API was rather broken: The initial call could raise a `ChipStackError`, but the overall commissioning result was returning a `PyChipError`. This all changed with this PR https://github.com/project-chip/connectedhomeip/pull/33905.